packages/react-components/src/form/tests/chromatic/Form.chroma.jsx

```diff
@@ -130,7 +131,7 @@ stories()
             <div>
                 <Field>
                     <Label>Departure date</Label>
-                    <TextInput placeholder="AAAA/MM/DD" />
+                    <TextInput placeholder="dd/mm/yyyy" />
```

Comment:
Maybe you should also replace this one by a DateInput?

Reply:
Maybe change one of these for a DateRangeInput?

Reply:
Maybe you should also replace this one by a DateInput? I was doing it ;)

Good idea. Will update.